src/map/battle.cpp

```diff
@@ -5405,6 +5405,14 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 			if (sc && sc->getSCE(SC_GIANTGROWTH) && rnd()%100 < 30)
 				skillratio *= 2;
 			break;
+		case DK_DRAGONIC_BREATH:
+			//TODO: needs official HP/SP scaling [Muh]
+			skillratio += -100 + 50 + 350 * skill_lv + 5 * sstatus->pow;
+			skillratio += sstatus->max_hp / 500 + status_get_max_sp(src) / 40;
+			if (sc && sc->getSCE(SC_DRAGONIC_AURA))
+				skillratio += sstatus->max_hp / 500 + status_get_max_sp(src) / 40;
```

Comment:
```
Damage is increased or reduced depends on user's MaxHP and MaxSP.
When using under Dragonic Aura buff, increases damage rate from user's MaxHP and MaxSP.
```
This sounds for me like it would be adjusted twice.

Reply:
Looking at the current format of the skill, it looks like this is adjusted twice, so I think this is good now?

Reply:
Just a little note. I still don't know exact HP / SP scaling but:
- it's much stronger than here
- the extra boost from dragonic aura is not as strong as the base scaling (<50% of the base effect)